feature-libs/asm/customer-360/components/sections/asm-customer-customer-coupon/asm-customer-customer-coupon.component.html

```diff
@@ -0,0 +1,62 @@
+<cx-asm-customer-promotion-listing
+  [headerText]="'customer360.customerCoupons.headerText' | cxTranslate"
+  [emptyStateText]="
+    'customer360.customerCoupons.emptyDescription' | cxTranslate
+  "
+  [applyButtonText]="
+    'customer360.customerCoupons.applyButtonText' | cxTranslate
+  "
+  [applied]="'customer360.customerCoupons.applied' | cxTranslate"
+  [removeButtonText]="
+    'customer360.customerCoupons.removeButtonText' | cxTranslate
+  "
+  [entries]="entries$ | async"
+  [showRemoveButton]="true"
+  [showApplyButton]="true"
+  [isCustomerCoupon]="true"
+  (changeCustomerCouponTab)="changeTab($event)"
+  (searchCustomerCoupon)="searchCustomerCoupon($event)"
```

Comment:
seems no need this two events for tabs and input has put in asm-customer-customer-coupon component

Reply:
removed.